PAS_Database/dbo/Stored Procedures/USP_UpdateSOQPartCostDetails.sql

```diff
@@ -149,7 +150,7 @@ SET NOCOUNT ON
 						SET UnitSalesPriceExtended = (SELECT SUM(SOSC.UnitSalesPriceExtended) FROM DBO.SalesOrderQuoteStockLineCost SOSC WHERE SOSC.SalesOrderQuotePartId = @SalesOrderQuotePartId),
 						UnitCostExtended = (SELECT SUM(ISNULL(SOSC.UnitCostExtended, 0)) FROM DBO.SalesOrderQuoteStockLineCost SOSC WHERE SOSC.SalesOrderQuotePartId = @SalesOrderQuotePartId),
 						NetSaleAmount = (SELECT SUM(ISNULL(SOSC.NetSaleAmount, 0)) FROM DBO.SalesOrderQuoteStockLineCost SOSC WHERE SOSC.SalesOrderQuotePartId = @SalesOrderQuotePartId),
-						TotalRevenue = (SELECT SUM(ISNULL(SOSC.NetSaleAmount, 0)) + MiscCharges FROM DBO.SalesOrderQuoteStockLineCost SOSC WHERE SOSC.SalesOrderQuotePartId = @SalesOrderQuotePartId)
+						TotalRevenue = (SELECT SUM(ISNULL(SOSC.NetSaleAmount, 0)) + ISNULL(@Charges_S, 0) FROM DBO.SalesOrderQuoteStockLineCost SOSC WHERE SOSC.SalesOrderQuotePartId = @SalesOrderQuotePartId)
```

Comment:
Have you verified if we won't have any stockline and just charges added at part level? In that case @Charges_S will be NULL